functions.php

```diff
@@ -5,13 +5,7 @@
 function scripts_and_styles_method() {
   $templateuri = get_template_directory_uri();
 
-  if (WP_DEBUG) {
-    $javascriptLibrary = $templateuri . '/dist/js/library.js';
-    $javascriptMain = $templateuri . '/src/js/main.js';
-  } else {
-    $javascriptLibrary = $templateuri . '/dist/js/library.min.js';
-    $javascriptMain = $templateuri . '/dist/js/main.min.js';
-  }
+  $javascriptMain = $templateuri . '/dist/js/main.js';
```

Comment:
why remove this?

Reply:
cuz dist/js/main.js is the minified file, theres no min file this time. library is gone cuz is all in main

Reply:
does it generate a map?

Reply:
obvs 😎